dashboard/app/models/user.rb

```diff
@@ -271,9 +271,7 @@ def permission?(permission)
   # Revokes all escalated permissions associated with the user, including admin status and any
   # granted UserPermission's.
   def revoke_all_permissions
-    self.admin = nil
-    save(validate: false)
-
+    update_attribute(:admin, nil)
```

Comment:
I think [update_column](https://apidock.com/rails/v3.1.0/ActiveRecord/Persistence/update_column) would even be more appropriate here, as it wouldn't have the side effect of saving other modified attributes, and it still skips validation. Credit to @mehalshah for pointing this out.